### ISSUE TITLE
CLI allows to configure BGP neighbor assigned to BGP peer group

### ISSUE THREAD

**Issue by xschily** [MERAprojects/ops-cli#8]
CLI allows to configure BGP neighbor assigned to BGP peer group. Ex.:  

```
router bgp 1
neighbor pg1 peer-group
neighbor pg1 timers 40 120
neighbor 10.0.12.2 peer-group pg1
neighbor 10.0.12.2 timers 30 90
do show running-config
Output:
...
neighbor pg1 timers 40 120
neighbor 10.0.12.2 timers 30 90
...
```

Original quagga does not allow to change configuration for a neighbor included into peer-group. Neighbor configuration is overwritten by peer group configuration when neighbor is added to peer group. Otherwise it is unclear which configuration is actually applied, i.e. in BGP timers case neighbor configuration is shown in CLI but peer-group configuration is applied by BGPD.

**Comment by mfwlarionov** [comment]
BGP neighbor timers correction. Ops-cli issue#8.
https://github.com/open-switch/ops-cli/pull/2

**Comment by mfwlarionov** [comment]
BGP neighbor timers correction. Ops-cli issue#8.
Pull request: 
Bgp timers configuration #16
https://github.com/MERAprojects/ops-cli/pull/16

**Comment by ekgrish** [comment]
Pull request #17  
"Fix for issue#8: CLI allows to configure BGP neighbor assigned to BGP peer group"

**Comment by ekgrish** [comment]
Pull request #18 
"Fix for issue#8: CLI allows to configure BGP neighbor assigned to BGP peer group"
+ synchronization of peer timers with its peer-group timers

**Comment by ekgrish** [comment]
Pull request #21
